tests/common/fixtures/duthost_utils.py

```diff
@@ -72,3 +73,18 @@ def disable_route_checker_module(duthosts, rand_one_dut_hostname):
     duthost = duthosts[rand_one_dut_hostname]
     for func in _disable_route_checker(duthost):
         yield func
+
+@pytest.fixture(scope='module')
+def disable_fdb_aging(duthost):
+    """
+    Disable fdb aging by bcmcmd 'age 0'.
+    The original config will be recovered after running test.
+    """
+    cmd = "bcmcmd \'age {}\'"
```

Comment:
This is a broadcom platform specific command. It can't be executed on other platforms.

Reply:
Good catch. I will try to implement the fixture in another way.

Reply:
The fixture is rewrite. FDB aging is disabled by loading a config file with ```swssconfig```.
```
[
    {
        "SWITCH_TABLE:switch": {
            "ecmp_hash_seed": "0",
            "lag_hash_seed": "0",
            "fdb_aging_time": "0"
        },
        "OP": "SET"
    }
]
```
I think it's a universal method for all platforms.
And I also added a clear of ptfadapter's dataplane at the end of ```setup_fdb``` because I found that the queue inside testutil might be filled up if too many packets flood in.